src/modifiers.rs

```diff
@@ -179,7 +180,7 @@ pub struct Redirect(pub Url);
 impl Modifier<Response> for Redirect {
     fn modify(self, res: &mut Response) {
         let Redirect(url) = self;
-        res.headers.set(headers::Location(url.to_string()));
+        res.headers.insert(headers::LOCATION, url.to_string().parse().unwrap()); // TODO: error handling?
```

Comment:
Here and 10 lines below: Here, also, the strings should always be valid HeaderValues. Thus, no error handling necessary.

Reply:
`parse()` could error here since only visible ASCII characters are permitted:  https://docs.rs/hyper/0.12.7/hyper/header/struct.HeaderValue.html#method.from_str